pandas/_testing.py

```diff
@@ -977,8 +977,15 @@ def assert_interval_array_equal(left, right, exact="equiv", obj="IntervalArray")
     """
     _check_isinstance(left, right, IntervalArray)
 
-    assert_index_equal(left.left, right.left, exact=exact, obj=f"{obj}.left")
-    assert_index_equal(left.right, right.right, exact=exact, obj=f"{obj}.left")
+    kwargs = {}
+    if left._left.dtype.kind in ["m", "M"]:
+        # We have a DatetimeArray or TimedeltaArray
+        kwargs["check_freq"] = False
+
+    # TODO: `exact` keyword?
```

Comment:
Does this TODO needs to be solved first? It was there before, but you now removed it? (so is not being ignored?)

Reply:
i think we're ok without the keyword, will remove the comment